Lombiq.Hosting.Tenants.Management/Service/JsonConfigurationParser.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text.Json;
+
+namespace Lombiq.Hosting.Tenants.Management.Service;
+
+public class JsonConfigurationParser
+{
+    private readonly Dictionary<string, string> Data = new(StringComparer.OrdinalIgnoreCase);
+    private readonly Stack<string> Paths = new();
```

Comment:
Use underscores.

Reply:
This file is a copy of Microsoft.Extensions.Configuration.Json.JsonConfigurationFileParser. Sadly their recommended way of using this class is to copy it. [https://github.com/dotnet/runtime/issues/73946](https://github.com/dotnet/runtime/issues/73946)

Reply:
Why would we copy things like field names? Also, please comment this on the class too.

Reply:
But it looks like it was a copy of an older version.